### PR TITLE
Tag SPTK.jl v0.2.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [r9y9/SPTK.jl](https://github.com/r9y9/SPTK.jl)
Release: [v0.2.0](https://github.com/r9y9/SPTK.jl/releases/tag/v0.2.0)
Travis: [Travis Build Status](https://travis-ci.org/r9y9/SPTK.jl/builds/222293829)
Diff: [vs v0.1.2](https://github.com/r9y9/SPTK.jl/compare/4b3d400ff3606ff71bfe60cdbe50b21e8ae7f2e6...ce61022be7f00c092dbdb66d725390356b608d30)
`requires` vs v0.1.2: 
```diff
--- 0.1.2/requires
+++ 0.2.0/requires
@@ -1,3 +1,3 @@
-julia 0.4
+julia 0.5
 BinDeps
-Compat 0.8.4
+Compat 0.8.4
```
cc: @r9y9

### CODE REVIEW

File: SPTK/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+BinDeps
+Compat 0.8.4
```
Comment (tkelman) [inline SPTK/versions/0.2.0/requires]: if you're dropping 0.4 support, looks like you don't need Compat at all right now?

Comment (r9y9) [inline SPTK/versions/0.2.0/requires]: You are right. Fixed in my repository. Should I re-tag v0.2.0 or add a new tag v0.2.1 instead?

Comment (tkelman) [inline SPTK/versions/0.2.0/requires]: either would work, up to you

Comment (r9y9) [inline SPTK/versions/0.2.0/requires]: New PR: https://github.com/JuliaLang/METADATA.jl/pull/8867

Comment (tkelman) [inline SPTK/versions/0.2.0/requires]: want me to merge both?

Comment (r9y9) [inline SPTK/versions/0.2.0/requires]: Yes, please. Thank you!

**Pull request merged**